### ISSUE TITLE
Only one Workspace is selectable during ‘prisma init’

### ISSUE THREAD

**Issue by wes-cutting** [prisma/prisma#3149]
TL;DR : I would like to initialize a Prisma service in a second Workspace and am unable to select the region from CLI ‘prisma init’ process.

As I run through a new ‘prisma init’ I select the option ‘Demo server’ under the category ‘Or deploy to an existing Prisma server’. Before this process, I had logged in through the CLI and can see that it is linked to the right account as it shows the first Workspace I created. However, that is the only Workspace that is available from the CLI. So when it asks for me to select a region, I only see the first Workspace.

Is this intended behavior? If it isn’t, how do I reveal my other regions/workspaces so I can initialize a new service outside of my first environment cluster?

If more clarification is needed please let me know.

**Comment by timsuchanek** [comment]
![image](https://user-images.githubusercontent.com/1094804/46136858-c991a780-c248-11e8-8b79-bf9c41c2f665.png)

It's already enough information. When selecting the demo server, only the demo servers for one workspace are shown. But actually all workspaces should be available in the selection.

**Comment by divyenduz** [comment]
This fix is available in the stable version - 1.18.1 - `npm install -g prisma` 🎉
